regulations/static/regulations/js/source/app-init.js

```diff
@@ -25,6 +26,7 @@ Backbone.$ = $;
     init: function() {
         Router.start();
         this.bindEvents();
+        $('table').stickyTableHeaders();
```

Comment:
On the initial page load, the main view does not get rendered - hence loaded is not called in that situation.  I don't like that this is in two places - but I feel for the BB stuff the loaded() function is where this code belongs.